src/Evolution/Systems/CurvedScalarWave/Worldtube/RadiusFunctions.cpp

```diff
@@ -0,0 +1,56 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Evolution/Systems/CurvedScalarWave/Worldtube/RadiusFunctions.hpp"
+
+#include <cmath>
+#include <cstddef>
+
+#include "Utilities/ErrorHandling/Error.hpp"
+
+namespace CurvedScalarWave::Worldtube {
+
+double smooth_broken_power_law(double orbit_radius, double alpha, double amp,
+                               double rb, double delta) {
+  if (alpha < 0. or alpha > 4.) {
+    ERROR(
```

Comment:
Factor out these two checks into a `detail` namespace so they share the same wording for both this function and the deriv